internal/resolvers/youtube/resolver.go

```diff
@@ -19,23 +20,35 @@ import (
 )
 
 const (
-	youtubeTooltip = `<div style="text-align: left;">
+	youtubeVideoTooltip = `<div style="text-align: left;">
 <b>{{.Title}}</b>
 <br><b>Channel:</b> {{.ChannelTitle}}
 <br><b>Duration:</b> {{.Duration}}
 <br><b>Published:</b> {{.PublishDate}}
 <br><b>Views:</b> {{.Views}}
 <br><span style="color: #2ecc71;">{{.LikeCount}} likes</span>&nbsp;•&nbsp;<span style="color: #e74c3c;">{{.DislikeCount}} dislikes</span>
 </div>
+`
+
+	youtubeChannelTooltip = `<div style="text-align: left;">
+<b>{{.Title}}</b>
+<br><b>Publish Date:</b> {{.PublishDate}}
+<br><b>Subscribers:</b> {{.Subscribers}}
+<br><b>Views:</b> {{.Views}}
+</div>
 `
 )
 
 var (
-	videoCache = cache.New("youtube_videos", load, 24*time.Hour)
+	videoCache = cache.New("youtube_videos", loadVideos, 24*time.Hour)
+	channelCache = cache.New("youtube_channels", loadChannels, 24*time.Hour)
```

Comment:
Could you add a comment here for how different link types are cached?

Reply:
Commented.